aws/PAS-AIO-DR-template.yaml

```diff
@@ -0,0 +1,1227 @@
+Resources:
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 30
+    DeletionPolicy: Retain
+  VaultCfnInitLogStream:
+    Type: AWS::Logs::LogStream
+    Properties: 
+      LogGroupName: !Sub '${LogGroup}'
+      LogStreamName: 'VaultMachine/CfnInitLog'
+    DeletionPolicy: Retain
+  VaultInitLogStream:
+    Type: AWS::Logs::LogStream
+    Properties: 
+      LogGroupName: !Sub '${LogGroup}'
+      LogStreamName: 'VaultMachine/VaultInitLog'
+    DeletionPolicy: Retain
+  VaultPostInstallLogStream:
```

Comment:
i didn't see this log in Full-PAS or in AIO or SingleVault, do we need it there as well?

Reply:
exist in AIO

Reply:
can you please check with Liora about Full-PAS? and with Avishay about Single-Vault?